copyright

```diff
@@ -386,6 +386,21 @@ Copyright: Copyright Mike Koenig
 License: CC-BY-SA-3.0
 Comment: Taken from http://soundbible.com/1467-Grenade-Explosion.html
 
+Files: sounds/missile hit.wav
+Copyright: Copyright "Nbs Dark"
+License: Public Domain
+Comment: Taken from https://freesound.org/people/Nbs%20Dark/sounds/94185/
+
+Files: sounds/meteor.wav
+Copyright: Copyright "18hiltc"
+License: CC-BY-3.0
```

Comment:
This isn't currently one of the licenses used in the game, which would require pasting the license's whole text in this file and in other places where license info is tracked. But, since you're creating a derivative work from that sound, you could also just say that the derivative version is CC-BY-SA-3.0, which *is* a license used for other game assets.

Reply:
Fixed.